capstone/src/main/java/com/google/sps/servlets/VoterServlet.java

```diff
@@ -13,22 +15,31 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // String inputID = request.getParameter("inputID");
-    // String address = request.getParameter("address");
-    // String url = "https://www.googleapis.com/civicinfo/v2/voterinfo";
-    // String charset = "UTF-8";
-    // String query =
-    //     String.format(
-    //         "address=%s&inputID=%s",
-    //         URLEncoder.encode(address, charset), URLEncoder.encode(inputID, charset));
-    // URLConnection connection = new URL(url + "?" + query).openConnection();
-    // connection.setRequestProperty("Accept-Charset", charset);
-    // InputStream result = connection.getInputStream();
+    String electionId = request.getParameter("electionId");
+    String address = request.getParameter("address");
+    Dotenv dotenv = Dotenv.configure().directory(System.getProperty("user.dir")).load();
+    String key_prod = dotenv.get("API_KEY");
+    String Key_Test = "";
```

Comment:
This variable never gets set. Do we need it for this to run? If not, delete this and the logic below that sets it to key_prod & just use key_prod.

Reply:
Key prod is supposed to have restrictions that prevent us from deploying the code locally which is why we have key test for local testing. Before I push to github I take the test API out so it uses key prod instead. I don't know if it makes sense

Reply:
I think there might still be some confusion here, but we can wait for Chen to weigh in and in the meantime, you can go ahead and submit.